addon/templates/components/polaris-callout-card.hbs

```diff
@@ -14,10 +14,14 @@
       {{/polaris-text-container}}
 
       <div class="Polaris-CalloutCard__Buttons">
-        {{#with (component "polaris-button"
-          text=primaryAction.text
-          onClick=(action primaryAction.onAction))
-          as |primaryButton|
+        {{!-- template-lint-disable attribute-indentation --}}
+        {{!-- TODO remove rule above once linter handles this correctly --}}
+        {{#with
+          (component "polaris-button"
+            text=primaryAction.text
+            onClick=(action primaryAction.onAction)
+          )
+        as |primaryButton|
```

Comment:
This looks like it should be indented IMO... any thoughts?

Reply:
this seems to be what the linter wants....it complains otherwise

Reply:
😞 

Is that something that's configurable?

Reply:
(not saying it needs to be done now 😉)

Reply:
for now, I'm gonna fix, since we disabled this rule manually above with that comment...might be a glitch with the linter (didn't checked yet if there's any related issue)